tiledb/sm/query/writer.h

```diff
@@ -121,6 +121,13 @@ class Writer {
    */
   std::vector<std::string> attributes() const;
 
+  /**
+   * Fetch AttributeBuffer for attribute
+   * @param attribute to fetch
+   * @return AttributeBuffer for attribute
+   */
+  AttributeBuffer buffer(const std::string& attribute);
```

Comment:
This (and in `Reader`) can be `const` I believe, then the `const_cast` can be removed as well.

Reply:
Good catch, forgot to port that same change over from the subarray work. Pushed new commit with it const.